hw/dv/data/vcs/vcs.hjson

```diff
@@ -34,7 +34,7 @@
 
   // Defaults for VCS
   cov_metrics:          "line+cond+fsm+tgl+branch+assert"
-  vcs_cov_hier:         ""
+  vcs_cov_hier:         "-cm_hier {proj_root}/hw/dv/tools/vcs/cover.cfg"
```

Comment:
Need to find a way to override this variable if user desires to change this 'default' to something else.

Reply:
Agree. Will figure it out later.